support/cas-server-support-aws-cognito-authentication/src/main/java/org/apereo/cas/authentication/AmazonCognitoAuthenticationAuthenticationHandler.java

```diff
@@ -114,4 +124,12 @@ protected AuthenticationHandlerExecutionResult authenticateUsernamePasswordInter
             throw new FailedLoginException(e.getMessage());
         }
     }
+
+    public Map<String, String> getMappedAttributes() {
```

Comment:
Remove

Reply:
I will remove it. I see that it is lot cleaner to do the transformation inside the handler.